Assets/Shopify/UIToolkit/ShopControllers/Classes/CartController.cs

```diff
@@ -84,7 +84,7 @@ public class CartItemsChangeEvent : UnityEvent<List<CartItem>> {}
         /// </summary>
         /// <param name="variant">The variant to modify</param>
         /// <param name="quantity">The desired quantity</param>
-        public void UpdateVariant(ProductVariant variant, int quantity) {
+        public void UpdateVariant(ProductVariant variant, long quantity) {
```

Comment:
The Shopify SDK uses `long`s for representing quantities so instead of flooring it down I just updated the method to use `int`.

Reply:
lol sounds good but who would ever order > `int.MaxValue` products in a cart hahaha

Reply:
Maybe someone _really_ likes doritoes who knows? I think it's just a GraphQL/API thing.